city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,111 @@
+        for title in response.css(".field--name-field-generic-body h3"):
+            full_str = title.xpath(".//text()").extract()[0]
+            date_match = re.search(r"[a-z]{3,9} \d{1,2},? \d{4}", full_str, flags=re.I).group(0)
+            date_obj = datetime.strptime(re.sub(',', '', date_match), '%B %d %Y')
+            new_date_obj = datetime(date_obj.year, date_obj.month, date_obj.day, 13, 00)
+            for one_day_links in title.xpath('.//following-sibling::ul[1]/li'):
+                href = one_day_links.xpath('.//@href').get()
+                name = one_day_links.xpath('.//text()').extract()
+                if href is None:
+                    continue
+                else:
+                    link = response.urljoin(href)
+                self.link_date_map.setdefault(new_date_obj, []).append({
+                    "title": "".join(name),
+                    "href": link
+                })
+        yield response.follow(
+            "https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings",
+            callback=self._parse_detail
+        )
+
+    def _parse_detail(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]'
+        )
+        year_list = items_list.xpath('.//h2')
+        for years in year_list:
+            year = re.search(r'\d{4}', years.extract()).group(0)
+            for items in years.xpath('.//following-sibling::ul[1]/li'):
+                item = items.extract()
+                start = self._parse_start(items, year)
+                meeting = Meeting(
+                    title=self._parse_title(items),
+                    description='',
+                    classification=self._parse_classification(item),
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="See agenda for meeting time",
+                    location=self._parse_location(item),
+                    links=self.link_date_map.get(start, []),
+                    source=response.url,
+                )
+                if "Rescheduled" in meeting["title"]:
+                    meeting["status"] = "Rescheduled"
+                    meeting["title"] = re.sub(r"Rescheduled", "", meeting["title"]).strip()
+                else:
+                    meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        title_obj = item.xpath(".//text()").extract()
+        title = re.sub(r'\(.+\)', '', "".join(title_obj))
+        title = title.split("-")[-1].strip(" ")
```

Comment:
The output of this line can be returned instead of assigning it to `title` again if it's returned in the next line